entities/entities/money_printer/init.lua

```diff
@@ -75,13 +75,13 @@ end
 
 function ENT:Fireball()
     if not self:IsOnFire() then self.burningup = false return end
-    local dist = math.random(20, 280) -- Explosion radius
+    local dist = math.random(400, 78400) -- Explosion radius (20, 280) ^ 2
     self:Destruct()
     for k, v in pairs(ents.FindInSphere(self:GetPos(), dist)) do
         if not v:IsPlayer() and not v:IsWeapon() and v:GetClass() ~= "predicted_viewmodel" and not v.IsMoneyPrinter then
             v:Ignite(math.random(5, 22), 0)
         elseif v:IsPlayer() then
-            local distance = v:GetPos():Distance(self:GetPos())
+            local distance = v:GetPos():DistToSqr(self:GetPos())
             v:TakeDamage(distance / dist * 100, self, self)
```

Comment:
This might cause a problem. Dividing two squared numbers does not give the same result as two normal numbers. It's a change of behaviour. `10/2 = 5, 100 / 4 = 25`